e2e/path/zonefile

```diff
@@ -66,3 +60,59 @@ _redirect.host-regex.host.path.example.com.            IN TXT "v=txtv0;type=host
+_redirect.2.predefined-version.host.path.example.com.       IN TXT "v=txtv0;type=host;to=https://second-record.host.path.example.com/docs/{1};re=^/(.*)$"
+
+predefined-versionword.host.path.example.com.               IN A 172.20.10.2
+_redirect.predefined-versionword.host.path.example.com.     IN TXT "v=txtv0;type=path;root=https://index.host.path.example.com;re=record"
+_redirect.1.predefined-versionword.host.path.example.com.   IN TXT "v=txtv0;type=host;to=https://first-record.host.path.example.com/release-{2}/examples{3};re=^/v([0-9]+\.[0-9]+)(/.*)?$"
+_redirect.2.predefined-versionword.host.path.example.com.   IN TXT "v=txtv0;type=host;to=https://second-record.host.path.example.com/{1};re=^/(.*)$"
+
+predefined-simpletospecific.host.path.example.com.             IN A 172.20.10.2
+_redirect.predefined-simpletospecific.host.path.example.com.   IN TXT "v=txtv0;type=path;root=https://index.host.path.example.com;re=record"
+_redirect.1.predefined-simpletospecific.host.path.example.com. IN TXT "v=txtv0;type=host;to=https://first-record.host.path.example.com/{2}/{3};re=^/([^/]*)/(.*)$"
+_redirect.2.predefined-simpletospecific.host.path.example.com. IN TXT "v=txtv0;type=host;to=https://second-record.host.path.example.com/{2};re=^/([^/]*)/?$"
+
+predefined.host.path.example.com.             IN A 172.20.10.2
+_redirect.predefined.host.path.example.com.   IN TXT "v=txtv0;type=path;root=https://index.host.path.example.com;re=record"
+_redirect.1.predefined.host.path.example.com. IN TXT "v=txtv0;type=host;to=https://predefined-regex.host.path.example.com/1;re=^/api-review$"
+_redirect.2.predefined.host.path.example.com. IN TXT "v=txtv0;type=host;to=https://predefined-regex.host.path.example.com/2;re=^/bot-commands$"
+_redirect.3.predefined.host.path.example.com. IN TXT "v=txtv0;type=host;to=https://predefined-regex.host.path.example.com/3;re=^/github-labels$"
+_redirect.4.predefined.host.path.example.com. IN TXT "v=txtv0;type=host;to=https://predefined-regex.host.path.example.com/4;re=^/good-first-issue$"
+_redirect.5.predefined.host.path.example.com. IN TXT "v=txtv0;type=host;to=https://predefined-regex.host.path.example.com/5;re=^/help-wanted$"
+_redirect.6.predefined.host.path.example.com. IN TXT "v=txtv0;type=host;to=https://predefined-regex.host.path.example.com/6;re=^/needs-ok-to-test$"
+_redirect.7.predefined.host.path.example.com. IN TXT "v=txtv0;type=host;to=https://predefined-regex.host.path.example.com/7;re=^/oncall$"
+_redirect.8.predefined.host.path.example.com. IN TXT "v=txtv0;type=host;to=https://predefined-regex.host.path.example.com/8;re=^/oncall-hotlist$"
+
+predefined-release.host.path.example.com.             IN A 172.20.10.2
+_redirect.predefined-release.host.path.example.com.   IN TXT "v=txtv0;type=path;root=https://index.host.path.example.com;re=record"
+_redirect.1.predefined-release.host.path.example.com. IN TXT "v=txtv0;type=host;to=https://predefined-regex.host.path.example.com/{2}{3};re=^/([^/]*)(/.*)?$"
```

Comment:
Why use a numbered predefined regex here?

Reply:
We can't use wildcards because there's a difference between how we extract the matches from the regex result.
In here we're fetching the first item from the matches:
https://github.com/txtdirect/txtdirect/blob/dcb70326ffb529872f4c77bdce3dfd8747f539d0/path.go#L265
But in predefined regex we're fetching all of the matches:
https://github.com/txtdirect/txtdirect/blob/dcb70326ffb529872f4c77bdce3dfd8747f539d0/path.go#L159

The first code is from 2 years ago and I can't really remember the reasoning behind it.

Reply:
Mhh that feels like something we should figure out and fix or document. Happy for you to create an issue and assign this to me or you :)